modules/gltf/src/gltf/gltf-builder.js

```diff
@@ -33,14 +36,6 @@ export default class GLTFBuilder extends GLBBuilder {
   //   throw new Error('Not yet imlemented');
   // }
 
-  // Add an extra application-defined key to the top-level data structure
-  // By default packs JSON by extracting binary data and replacing it with JSON pointers
-  addApplicationData(key, data, packOptions = {}) {
```

Comment:
I think we should keep this method here, it is part of the full "glTF builder API" we are offering. Spreading these methods out over two classes has just made it harder for maintainers to get the full picture of the API we provide.

We can rename the method in glbBuilder to e.g. `setJSONKey` or something with more low-level feeling.